scripts/IntentData.py

```diff
@@ -12,16 +12,23 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+from __future__ import absolute_import
 
 import re
-from wawCommons import eprintf
+from .wawCommons import eprintf
 from collections import OrderedDict
 
 '''
 Created on Jan 15, 2018
 @author: alukes
 '''
 
+try:
+    unicode        # Python 2
```

Comment:
There is no `unicode` import.

Reply:
__unicode__ is used on line 152.  Which is ok in Python 2 where __unicode__ is a builtin (no import required) but is not ok in Python 3 where __unicode__ was removed.